modules/test/playwright/pages/product-navigation-applications-menu/ApplicationsMenuPage.ts

```diff
@@ -8,12 +8,15 @@ import {Locator, Page, expect} from '@playwright/test';
 import {HomePage} from '../portal-web/HomePage';
 
 export class ApplicationsMenuPage {
+	readonly applicationMenuButton: Locator;
 	private readonly applicationsMenuTabButton: Locator;
 	private readonly clientExtensionsLink: Locator;
 	private readonly controlPanelButton: Locator;
 	private readonly dataMigrationCenterMenuItem: Locator;
+	readonly dataSetManagerMenuItem: Locator;
 	private readonly homePage: HomePage;
 	private readonly instanceSettingsLink: Locator;
+	readonly objectsLink: Locator;
 	private readonly objectsMenuItem: Locator;
 	readonly page: Page;
```

Comment:
Does it make sense to make `page` public? I'd say nobody should access the base `Page` from here, but have it instead injected by Playwright in the test...

Reply:
probably an oversight